### PR TITLE
Added the zenodo file with the contributors

### PR DESCRIPTION
**Pull request by nkorinek**
Added a zenodo file with all the contributors listed in this issue: https://github.com/earthlab/abc-classroom/issues/252

### CODE REVIEW

File: .zenodo.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      "name": "Wasser, Leah",
+      "orcid": "0000-0002-8177-6550"
+    },
+    {
+      "affiliation": "Biological Informatics Centre of Excellence Agriculture and Agri-food Canada",
+      "name": "Cranston, Karen",
+      "orcid": "0000-0002-4798-9499"
+    },
+  ],
+  "contributors": [
```
Comment (lwasser) [inline .zenodo.json:14]: @nkorinek can you please list time as a creator as well? he did originally create this package so i think it's important to list him there.

**Review by lwasser — CHANGES_REQUESTED** [review]
just one small change @nkorinek  and we can merge this!

**Comment by nkorinek** [comment]
@lwasser moved tim!

**Comment by lwasser** [comment]
thank you again @nkorinek !!

**Review by lwasser — APPROVED** [review]

**Pull request merged**